bzt/modules/selenium.py

```diff
@@ -864,26 +827,16 @@ def gen_assertion(self, assertion_config):
 
             if regexp:
                 assert_method = "self.assertEqual" if reverse else "self.assertNotEqual"
-                assertion_elements.append(self.gen_method_statement('re_pattern = re.compile("%s")' % val))
+                assertion_elements.append(self.gen_statement('re_pattern = re.compile("%s")' % val))
```

Comment:
We should probably use `repr()` (`%r`) instead of `str()` here, because `str` doesn't do character unescaping, while `repr` does.

This applies to Locust generator too.